matsim/scenariogen/data/__init__.py

```diff
@@ -327,16 +332,24 @@ class Trip:
     purpose: Purpose
     sd_group: SourceDestinationGroup
     valid: bool
+    dep_district: str = ""
+    arr_district: str = ""
+    arrival: int = 0
```

Comment:
There is departure and duration, is arrival necessary ?

Reply:
Technically not, but does is matter where the calculation of arrival is made? If it is not included as a field here, the calculation will be done in the next sampling step, now it is done here. If you insist I can change it, but to me it makes no difference.

Reply:
Basically, you would store redundant information in this class and potential csv files, because these classes are converted into tables often. Also, if you are not careful you might produce inconsistency between arrival, departure and duration.  In general, I would say the safer option is just to not add it as a field explicitly.

Reply:
Ok, I see. I'll remove the field then.